stm32_part_table.yaml

```diff
@@ -583,3 +583,26 @@ stm32g4:
     members:
       - STM32G474
       - STM32G484
+
+stm32mp1:
+  stm32mp151:
+    url: https://www.st.com/en/microcontrollers-microprocessors/stm32mp151.html
+    rm: RM0441
+    rm_title: STM32MP151
+    rm_url: https://www.st.com/resource/en/reference_manual/dm00366349-stm32mp151-advanced-armbased-32bit-mpus-stmicroelectronics.pdf
+    members:
+      - STM32MP151
+  stm32mp153:
+    url: https://www.st.com/en/microcontrollers-microprocessors/stm32mp153.html
+    rm: RM0442
+    rm_title: STM32MP153
+    rm_url: https://www.st.com/resource/en/reference_manual/dm00366355-stm32mp153-advanced-armbased-32bit-mpus-stmicroelectronics.pdf
+    members:
+      - STM32MP153
```

Comment:
We shouldn't add these to the parts table until they're included in the PAC; could you delete them for now and we'll re-add when device files are added for them?